### PR TITLE
Add shell script to generate certificates + update README.md to use openssl CLI

### PR DESCRIPTION
**Pull request by agambier**
Modifications added : 

- Rename app name to ssl-2way to prevent misunderstanding with the official openssl CLI.
- Add shell script to generate certificates & private keys.
- Update `README.md` so official openssl CLI can be used as the server or the client to validate the example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-A simple example program that demonstrates two-way authentication between a client
-and the server.
+A simple example program that demonstrates two-way authentication between a client and the server.
 
 There are a couple of things to be noted here:
 
 1. Hostname verification is not performed. If you want to perform it, you should take a look at [this repository](https://github.com/iSECPartners/ssl-conservatory).
 2. Don't plugin this code directly into multi-threaded applications, you need to call some additional routines so that OpenSSL routines become reentrant.
-3. You can generate the keys by looking at [this gist](https://gist.github.com/zapstar/4b51d7cfa74c7e709fcdaace19233443).
+3. You can generate the keys by looking at [this gist](https://gist.github.com/zapstar/4b51d7cfa74c7e709fcdaace19233443) or by using the included script `certs_gen.sh` (Uniy based OS).
```
Comment (zapstar) [inline README.md]: typo: `Uniy` instead of `Unix`

File: certs_gen.sh [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      -CA "${CA_CERT}" -CAkey "${CA_KEY}" \
+      -CAcreateserial -out "${CLIENT_CERT}" > /dev/null 2>&1
+  [ $? -ne 0 ] && echo "FAILED" && exit 1
+
+  echo "DONE"
+else
+  echo "SKIPPED"
+fi
+
+exit 0
```
Comment (zapstar) [inline certs_gen.sh]: Please add a newline in the end.

**Review by zapstar — CHANGES_REQUESTED** [review]

**Comment by zapstar** [comment]
Thank you very much for the PR!

File: README.md (hunk 1 above)
Comment (agambier) [reply README.md]: done

File: certs_gen.sh (hunk 2 above)
Comment (agambier) [reply certs_gen.sh]: done

**Review by zapstar — APPROVED** [review]

**Pull request merged**